CHANGELOG.md

```diff
@@ -10,7 +10,10 @@
 
 #### Enhancements
 
-* None.
+* Inline test failure messages to make development of SwiftLint easier. Test failures in triggering and non-triggering examples will appear inline in their respective files so you can immediately see which cases are working and which are not.  
+  [ZevEisenberg](https://github.com/ZevEisenberg)
+  [#3040](https://github.com/realm/SwiftLint/pull/3040)
```

Comment:
Can just drop this line completely: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Reply:
I'm confused. That page says:
>4. A list of Markdown hyperlinks to the issues the change addresses. One entry per line. Usually just one. If there was no issue tracking this change, you may instead link to the change's pull request.
Many other changelog entries link to the issue or PR. Why drop this line?